vimscript/stalinsort.vim

```diff
@@ -0,0 +1,15 @@
+function! s:StalinSort(list)
+  if empty(a:list)
+    return []
+  endif
+  let l:return = [a:list[0]]
+  for x in a:list
+    if x > l:return[-1]
```

Comment:
what is the result of a list with repeated numbers like `1 3 3 2 4`?

Reply:
It actually outputs [1, 3, 4]

Reply:
can you change it to output `1 3 3 4`?

Reply:
Done in the last commit :)